core/domain/services/graphql/types/Event.php

```diff
@@ -79,22 +73,22 @@ public function getFields()
                 esc_html__('Date/Time Event Created', 'event_espresso')
             ),
             new GraphQLOutputField(
-                'manager',
-                'User',
-                null,
-                esc_html__('Event Manager', 'event_espresso')
+                'dbId',
+                ['non_null' => 'Int'],
+                'ID',
+                esc_html__('The event ID.', 'event_espresso')
             ),
-            new GraphQLInputField(
-                'manager',
+            new GraphQLField(
+                'defaultRegStatus',
                 'String',
```

Comment:
IMHO, we should allow a limited set of statuses by using enum instead of string.

Reply:
great suggestion... will implement, thnx